src/platforms/android/usage/advanced-usage.mdx

```diff
@@ -260,6 +260,23 @@ If you want to set context data in the Java layer, then receive that context dat
 </application>
 ```
 
+#### Multi-Process Apps
+
+Due to using `ContentProvider`, Sentry Android SDK will not automatically initialize in non-main processes.
+
+If you'd like to automatically `auto-init` the Sentry Android SDK in other processes, add this to your manifest:
```

Comment:
```suggestion
The Sentry Android SDK will not automatically initialize in non-main processes because of `ContentProvider`. If you'd like to automatically `auto-init` the Sentry Android SDK in other processes, add the following to your manifest: 
```